requirements.txt

```diff
@@ -1,6 +1,6 @@
-h5py==2.9.0
+h5py==3.6.0
```

Comment:
Let's not change this. Don't see a need.

Reply:
Agreed.